migrations/2017_12_10_094104_create_rooms_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateRoomsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('rooms', function (Blueprint $table) {
```

Comment:
`groups`, not `rooms`. I feel `group` is more generic than a `room`. In the primary school context, I think a _groep_ is more intuitive than a room (_kamer_, _klaslokaal_?)

Reply:
Personally, I think a `group` looks a lot like `team`, that's why I chose for `room`. Moreover, I expected players to be in the same room while playing the game, but I don't know if that is true.

Reply:
Yeah makes sense. I was mostly thinking about how we want to present this in the interface already, as _group name_  or _room name_. I fear this may invite a literal interpretation by teachers to fill-in something like 2B, or 1A or something.

We should make sure that does not happen: maybe we can auto-generate a group name? E.g. `new room` and then just generate a 4-char alphanumeric sequence of sorts?

Reply:
So, I will keep `room`.

`E.g. a new room button and then just generate a 4-char alphanumeric sequence of sorts?`
Yes, that is indeed an option.